description/active/universal/15 Credits.md

```diff
@@ -55,32 +54,33 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
 - CatOnStilts
 - Cosmog
 - Deadinsky
-- FroggiestSpirit
+- FroggestSpirit
 - GACT
 - GlaceonMyst
 - Gronnoct
 - Kelcyus
 - Legtendga
 - Mapper
 - MegamanOmega
+- Mmmmmmmmmmmmmmmmm
 - Moonbyes
+- Monstarules
 - MyRattata
-- Mmmmmmmmmmmmmmmmm
 - NonsensicalDreamer
 - Phaydra_
-- Pia Carrot
+- PIACarrot
```

Comment:
Didn't you say PIA wanted to be temporarily removed? https://discord.com/channels/927292379436814406/928400707269320734/975532546383306762

Reply:
Talked with him again, he's okay with remaining there now. It was a time-sensitive thing before we decided I would update the description.